impexp-config/src/main/java/org/citydb/config/project/exporter/ComputeNumberMatched.java

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
Should this class be moved to the `config.project.common` package? Because the CityGML deleter module can also make use of it.  
https://github.com/3dcitydb/importer-exporter/blob/master/impexp-core/src/main/java/org/citydb/core/operation/deleter/controller/Deleter.java#L256

Reply:
Indeed, good point. Will make a proposal.

Reply:
addressed in e06d8cd20b0fb047872690a906d69e7717e30dc9 and 6e1787e044610e8317f524bc6f0ecc3357b59414